code/datums/datacore.dm

```diff
@@ -35,12 +35,17 @@
 		var/rank = t.fields["rank"]
 		var/real_rank = t.fields["real_rank"]
 		if(OOC)
-			var/active = 0
-			for(var/mob/M in GLOB.player_list)
-				if(M.real_name == name && M.client && M.client.inactivity <= 10 * 60 * 10)
-					active = 1
+			var/activetext = "Inactive"
+			for(var/mob/M in GLOB.human_list)
```

Comment:
I recommend doing a typeless loop here.

Reply:
Thanks Steel, I can't believe I did not notice that.